torch/distributed/rpc/internal.py

```diff
@@ -58,6 +67,14 @@ def serialize(self, obj):
         p = pickle.Pickler(f)
         p.dispatch_table = self._dispatch_table
 
+        # rpc api could accept user picklers inheriting from _InternalRPCPickler to serialize rref,
+        # user picklers could have different initialization function from _InternalRPCPickler,
+        # but all the user picklers should call serialize() and use _rref_reducer to pickle rref
+        # in python. also, when _internalRPCPickler is imported to rpc/api.py, rpc.RRef is not 
```

Comment:
This is call `_internal_rpc_pickler` in the code

Reply:
Or do you mean the `_InternalRPCPickler ` class? We didn't import the class type in `rpc/api.py`, no?

Reply:
will change

Reply:
no, we did not import class type, just import the global pickler